common/script/ops/openMysteryItem.js

```diff
@@ -1,32 +1,41 @@
 import content from '../content/index';
+import i18n from '../i18n';
+import {
+  BadRequest,
+} from '../libs/errors';
+
+module.exports = function openMysteryItem (user, req = {}, analytics) {
+  let item = user.purchased.plan.mysteryItems.shift();
 
-module.exports = function(user, req, cb, analytics) {
-  var analyticsData, item, ref, ref1;
-  item = (ref = user.purchased.plan) != null ? (ref1 = ref.mysteryItems) != null ? ref1.shift() : void 0 : void 0;
   if (!item) {
-    return typeof cb === "function" ? cb({
-      code: 400,
-      message: "Empty"
-    }) : void 0;
+    throw new BadRequest(i18n.t('mysteryItemIsEmpty', req.language));
   }
-  item = content.gear.flat[item];
+
+  item = Object.assign(content.gear.flat[item]);
```

Comment:
as far as I know Object.assign works with 2 arguments not one (see https://developer.mozilla.org/it/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) , also I'd use lodash's cloneDeep here in case `item` has some nested objects (or if they get added later)